tools/UpdateDotnetRuntime.ps1

```diff
@@ -0,0 +1,130 @@
+[CmdletBinding()]
+param (
+)
+
+function Update-GlobalJson([string] $Version) {
```

Comment:
a short description of each function would be good

Reply:
fixed